Lib/bdb.py

```diff
@@ -194,6 +277,12 @@ def user_exception(self, frame, exc_info):
         pass
 
     def _set_stopinfo(self, stopframe, returnframe, stoplineno=0):
+        """Set the attributes for stopping.
+
+        If `stoplineno` is greater than or equal to 0, then stop at line
```

Comment:
In this doctstring you are wrapping the argument stoplineno with `, while in all the other doctstrings you are not. I think to be consistent you have to remove the `` around stoplineno.